Documentation/edgefs-cluster-crd.md

```diff
@@ -336,7 +336,7 @@ subjects:
   name: rook-edgefs-cluster
   namespace: rook-edgefs
 ---
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1beta2
```

Comment:
As written in the comments, going to `apps/v1` directly would be better as all Rook supported K8S versions support this stable API.

Reply:
done